files/rsyslog

```diff
@@ -2,9 +2,9 @@ RSYSLOGCUSTOMHEADER (?:(?:<%{NONNEGINT:syslog_abspri}>(?:%{NONNEGINT:syslog_vers
 
 # new style, needs custom logstash config
 RSYSLOG_SSHD %{RSYSLOGCUSTOMHEADER} %{IPORHOST:syslog_hostname} (?<program>sshd)(?:\[%{POSINT:pid}\])?: %{SSH_MESSAGE}
-RSYSLOG_MSG (?:%{MODULECMD_MSG}|%{NFS_MSG}|%{CEPH_MSG}|%{GREEDYDATA:syslog_message})
+RSYSLOG_MSG (?:%{MODULECMD_MSG}|%{NFS_MSG}|%{CEPH_MSG}|%{OPENNEBULA_MSG}|%{KEYVALUE_MSG}|%{GREEDYDATA:syslog_message})
```

Comment:
switch the keyvalue with opennebula, there must be somtheing open-ended in the ONE patterns that gobbles up the keyvalue stuff